client/ui/qml/Pages2/PageStart.qml

```diff
@@ -66,6 +67,7 @@ PageType {
             } else {
                 tabBarStackView.push(pagePath, { "objectName" : pagePath }, StackView.Immediate)
             }
+            tabBar.isServerInfoShow = page === PageEnum.PageSettingsServerInfo
```

Comment:
From this page we can go on to the various protocols/services configuration pages and protocols/services installation pages. When going to these pages, the "PageHome" icon is highlighted again